LandSandBoat/server
# Add Immanence functionality

**_I affirm:_**

## What does this pull request do?



- This pull request adds the necessary framework to handle magic skillchains from the use of the Scholar ability "Immanence" 

## Steps to test these changes



1.  Apply the changed files from this pull request to your local repository.
2.  Clean and rebuild the solution.
3.  Login character and load up a scholar with a level higher than level 87.
4.  Use Dark Arts > Strategems > Addendum Black
5.  Use Strategems > Immanence
6.  Cast elemental spell (I used Blizzard V)
7.  Use Strategems > Immanence
8.  Cast elemental T1 skillchain element (I used Water V)
9.  Observe skill chain (Mine was Fragmentation.

![image](https://github.com/LandSandBoat/server/assets/99501957/669a88ff-a90a-4c1a-a342-90a63976b9c6)
